Scripts/Source/UD_RandomRestraintManager.psc

```diff
@@ -80,32 +80,43 @@ bool _mutex = false
 ;14b= zad_DeviousGloves
 ;15b= zad_DeviousHood
 
+;adding missed items
+;16b= zad_DeviousCorset
+;17b= zad_DeviousHarness
+
 FormList Property UD_CheckKeywords auto
 FormList Property SuitableKeywords auto
 
 bool ready = false
 Event onInit()
+
     UD_CheckKeywords.revert()
-    UD_CheckKeywords.addForm(libs.zad_DeviousCollar)         ;iIndex = 0
-    UD_CheckKeywords.addForm(libs.zad_DeviousArmCuffs)        ;iIndex = 1
-    UD_CheckKeywords.addForm(libs.zad_DeviousBelt)            ;iIndex = 2
-    UD_CheckKeywords.addForm(libs.zad_DeviousBlindfold)        ;iIndex = 3
-    
-    UD_CheckKeywords.addForm(libs.zad_DeviousBra)            ;iIndex = 4
-    UD_CheckKeywords.addForm(libs.zad_DeviousBoots)            ;iIndex = 5
-    UD_CheckKeywords.addForm(libs.zad_DeviousGag)            ;iIndex = 6
-    UD_CheckKeywords.addForm(libs.zad_DeviousHeavyBondage)    ;iIndex = 7
-    
-    UD_CheckKeywords.addForm(libs.zad_DeviousLegCuffs)        ;iIndex = 8
-    UD_CheckKeywords.addForm(libs.zad_DeviousSuit)            ;iIndex = 9
-    UD_CheckKeywords.addForm(libs.zad_DeviousPlugVaginal)    ;iIndex = 10
-    UD_CheckKeywords.addForm(libs.zad_DeviousPlugAnal)        ;iIndex = 11
-    
-    UD_CheckKeywords.addForm(libs.zad_DeviousPiercingsVaginal)    ;iIndex = 12
-    UD_CheckKeywords.addForm(libs.zad_DeviousPiercingsNipple)    ;iIndex = 13
-    UD_CheckKeywords.addForm(libs.zad_DeviousGloves)            ;iIndex = 14
-    UD_CheckKeywords.addForm(libs.zad_deviousHood)                 ;iIndex = 15
-    
+
```

Comment:
Rearranging the keywords will break previously created saves for older UD version. My current policy is to make new version UD backward compatible. If you still want to implement it like this, you will have to add and hook Update function, so when the game is loaded, the array is wiped and filled with rearranged values. In that case, also, please update the documentation which is at top of the script

Reply:
Well, it has to be re-filled anyways because of added 2 keywords for corsets and harnesses.

I'm not sure how to add that Update function, any pointers for me?

Yep, will update the documentation so it will state clearly what is what. Makes total sense. I've left it in this state to be able to see "before" and "after" and assess changes.

Reply:
I think I figured out Update() function, added hook into main UD script, added necessary functions in UDRRM, tested in old save - it worked.

Cleaned up documentation, it's all written now as it should.